type-tests/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "types": []
```

Comment:
I added this to avoid global types leaking from `@types/*` packages.

Reply:
I would recommend keeping this file:

`types: []` makes the tests run faster. There is less to check for the compiler.

`@types/jest` define globals like `test()` or `describe()`. There would be no error if you forget to import them from `tstyche`, but those helpers are not collected by the TSTyche runner

some libraries ship `@types/*`. For instance, `jest` ships `@types/node`. People keep on complaining that these alter globals, but the only solution is to have `types: []` set

`@types/node` augments typings of `fetch()`, `ArrayBuffer`, something else. Can't check, I'm on mobile right now. TSTyche is only checking types, there is no execution. These altered types can lead to hard to debug issue.

In my opinion, `types: []` must be set by default. Magic global typings are not good.

---

TSTyche is using compiler options from `tsconfig.json` file nearest to a file in question which includes that file. If a file is not included, TSConfig is ignored and the algorithm walks up the file tree. If no matching TSConfig is found, default compiler options are set.

This is exactly how text editors resolve TSConfig. Only trouble: everyone sets different defaults.

So keeping the file makes sure that test files will be always included in some `tsconfig.json` explicitly.

These are more or less all the reasons I would recommend keeping the file. Fell free to remove it, if that sounds fine for you.

Reply:
Ah, thanks for the explanation! So it makes perfect sense to keep the file.

Reply:
Yes, indeed. 

If you leak Jest globals to you source files, those will throw at runtime then used. Type tests have no runtime. That's the main reason to isolate them as much as possible.

Reply:
I see. So using `@jest/globals` actually makes more sense than relying on `@types/jest`. I opened an issue about this, so I don't forget:

* https://github.com/FloEdelmann/vue-ts-types/issues/508